core/jazz_assets/config/global-config.json

```diff
@@ -35,8 +35,7 @@
     "iam_role",
     "sqs",
     "kinesis_stream",
-    "apigee_proxy",
-    "http"
+    "apigee_proxy"
```

Comment:
why are we removing http here?

Reply:
We dont need the http asset_type over here as we are capturing it as enpoint_url and apigateway